### DISCUSSION TITLE
Where to download `itp` file of `CDL2` for martini3?

### DISCUSSION THREAD

**Discussion by chaofan520** [Martini-Force-Field-Initiative/MartiniDiscussions#17] [Lipids]
Where to download `itp` file of `CDL2` for martini3?
`"CDL2": (moltype, "GL5 PO41 GL1 GL2 C1A C2A D3A C4A C5A   - C1B C2B D3B C4B C5B   - PO42 GL3 GL4 C1C C2C D3C C4C C5C   - C1D C2D D3D C4D C5D   -"), # Warning not the same names is in .itp`

Have no file in `https://cgmartini.nl/docs/downloads/force-field-parameters/martini3/lipidome.html`.

**Comment by chaofan520** [comment]
@ricalessandri

**Reply by ricalessandri** [reply]
Not sure anybody developed yet a cardiolipin model for Martini 3.

**Reply by ricalessandri** [reply]
@Lp0lp may know more than me

**Reply by chaofan520** [reply]
Thank you for your help!

**Comment by amiel-ab** [comment]
Hi, 
There are actually parameters for the cardiolipins in here:
https://pubs.acs.org/doi/10.1021/acs.jctc.3c00471
You'll find CDL1.itp you just have to change the charge in the 2nd phosphate to obtain CDL2 , you would not need to tune other parameters I believe.
Do you work with mitochondrial proteins by any chances ?

**Reply by chaofan520** [reply]
Thank you for your reply! 
I want to build a Outer Membrane of E.coli with CDL2. But i can not find the correct `itp` file of CDL2 for insane.
